Week1/Week1/ViewController.swift

```diff
@@ -0,0 +1,222 @@
+        }
+        
+        self.view.addSubview(performPlusButton)
+        performPlusButton.snp.makeConstraints{
+            $0.top.equalTo(textField2.snp.bottom).offset(Size.Margin.large)
+            $0.height.equalTo(Size.Area.large)
+            $0.leading.equalTo(self.view.safeAreaLayoutGuide).offset(Size.Margin.small)
+            $0.trailing.equalTo(self.view.safeAreaLayoutGuide).offset(-Size.Margin.small)
+        }
+        
+        self.view.addSubview(performMinusButton)
+        performMinusButton.snp.makeConstraints{
+            $0.top.equalTo(performPlusButton.snp.bottom).offset(Size.Margin.small)
+            $0.height.equalTo(Size.Area.large)
+            $0.leading.equalTo(self.view.safeAreaLayoutGuide).offset(Size.Margin.small)
+            $0.trailing.equalTo(self.view.safeAreaLayoutGuide).offset(-Size.Margin.small)
+        }
+        
+        self.view.addSubview(performMultipleButton)
+        performMultipleButton.snp.makeConstraints{
+            $0.top.equalTo(performMinusButton.snp.bottom).offset(Size.Margin.small)
+            $0.height.equalTo(Size.Area.large)
+            $0.leading.equalTo(self.view.safeAreaLayoutGuide).offset(Size.Margin.small)
+            $0.trailing.equalTo(self.view.safeAreaLayoutGuide).offset(-Size.Margin.small)
+        }
+        
+        self.view.addSubview(performDivideButton)
+        performDivideButton.snp.makeConstraints{
+            $0.top.equalTo(performMultipleButton.snp.bottom).offset(Size.Margin.small)
+            $0.height.equalTo(Size.Area.large)
+            $0.leading.equalTo(self.view.safeAreaLayoutGuide).offset(Size.Margin.small)
+            $0.trailing.equalTo(self.view.safeAreaLayoutGuide).offset(-Size.Margin.small)
+        }
+        
+        self.view.addSubview(resultLabel)
+        resultLabel.text = ""
+        resultLabel.snp.makeConstraints{
+            $0.top.equalTo(performDivideButton.snp.bottom).offset(Size.Margin.large)
+            $0.height.equalTo(Size.Area.large)
+            $0.leading.equalTo(self.view.safeAreaLayoutGuide).offset(Size.Margin.small)
+            $0.trailing.equalTo(self.view.safeAreaLayoutGuide).offset(-Size.Margin.small)
+        }
+    }
+    
+    private func makeLogics() {
+        performPlusButton.addTarget(self, action: #selector(onTapButton(_:)), for: .touchUpInside)
+        performMinusButton.addTarget(self, action: #selector(onTapButton(_:)), for: .touchUpInside)
+        performMultipleButton.addTarget(self, action: #selector(onTapButton(_:)), for: .touchUpInside)
+        performDivideButton.addTarget(self, action: #selector(onTapButton(_:)), for: .touchUpInside)
+    }
+    
+    @objc
+    func onTapButton(_ button: UIButton) {
+        print(button.tag)
+        // 1. textfield 변환 후 오류 식별
+        guard let text1 = textField1.text, let text2 = textField2.text else {
+            resultLabel.text = ErrorMessage.unknown
+            return
+        }
+        
+        if text1 == "" || text2 == "" {
```

Comment:
```suggestion
        if text1.isEmpty || text2.isEmpty {
```
isEmpty 사용하면 가독성면에서 좋을 것 같습니다~